### PR TITLE
GCC day2 overview

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

File: posts/2024/gcc_day2_programme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+media:
+  - mastodon
```
Comment (wm75) [inline posts/2024/gcc_day2_programme.md:3]:
```suggestion
  - mastodon-eu-freiburg
```

File: posts/2024/gcc_day2_programme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+hashtags:
+  mastodon:
```
Comment (wm75) [inline posts/2024/gcc_day2_programme.md:6]:
```suggestion
  mastodon-eu-freiburg:
```

File: posts/2024/gcc_day2_programme.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+media:
+  - mastodon-eu-freiburg
+
+hashtags:
+  mastodon-eu-freiburg:
+    - #UseGalaxy
```
Comment (wm75) [inline posts/2024/gcc_day2_programme.md:7]:
```suggestion
    - UseGalaxy
```

**Comment by wm75** [comment]
Ready if you like it @erxleben @bgruening

**Review by sanjaysrikakulam — APPROVED** [review]

**Comment by arash77** [comment]
The emoji `:flag-cz:` is not working on mastodon. Maybe it is best to use the emoji itself 🇨🇿.

**Pull request merged**